docs/apis/avalanchego/avalanchego-release-notes.md

```diff
@@ -6,6 +6,27 @@
 
 :::
 
+## v1.7.11 ([View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.7.11))
+
+This version is backwards compatible to [v1.7.0](https://github.com/ava-labs/avalanchego/releases/tag/v1.7.0). It is optional, but encouraged.
+
+**Networking**
+
+- Bandwidth reduction improvements:
+    - By default, non-validators now send only pull queries, not push queries.
+    - By default, validators now send both pull queries and push queries upon inserting a container into consensus. Previously, nodes sent only push queries.
+    - See new [flags](../../nodes/maintain/chain-config-flags.md) `snow-mixed-query-num-push-vdr` and `snow-mixed-query-num-push-non-vdr`.
+
+**Logging**
+- Changed AvalancheGo's internal logger with [uber-go/zap](https://github.com/uber-go/zap).
+- Changed AvalanceGo's log rotation with [lumberjack](https://github.com/natefinch/lumberjack).
+- Renamed `log-display-highlight` to `log-format` and added `json` option.
+- Added `log-rotater-max-size`, `log-rotater-max-files`, `log-rotater-max-age`, `log-rotater-compress-enabled` options for log rotation.
+
+**PlatformVM**
+- Added `subnetID` parameter to `platform.getTotalStake` API call.
+- Added `connected` field for subnet validators in `platform.getCurrentValidators` and `platform.getPendingValidators` API call.
```

Comment:
Do we need to update the doc on these two APIs?

Reply:
@ceyonur Can you please update the API docs? :pray:

Reply:
api docs already show these fields are available for all validators. what they do not specify is they were not available for subnet validators. but now they're available for subnet vdrs as well. so we fixed the docs by fixing the actual API :D. nevertheless going to emphasize that. thanks.